doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex

```diff
@@ -183,6 +183,475 @@ \subsubsection{Outputs}\label{outputs-015}
+
+
+For the dehumidification control modes, the maximum humidity setpoint on the Sensor Node is used. This must be set using a ZoneControl:Humidistat object. When extra dehumidification is required, the system may not be able to meet the humidity setpoint if its full capacity is not adequate. If the dehumidification control type is specified as CoolReheat, then the system may require reheat coil type and name elsewhere. Although the reheat coil is required only when CoolReheat is selected, the optional reheat coil may be present for any of the allowed Dehumidification Control Types.
+
+Valid humidity setpoint managers include:
+\begin{itemize}
+\item SetpointManager:SingleZone:Humidity:Maximum
+\item SetpointManager:MultiZone:Humidity:Maximum
+\item SetpointManager:MultiZone:MaximumHumidity:Average
+\item SetpointManager:OutdoorAirPretreat
+\end{itemize}
+
+\paragraph{Field: Run On Sensible Load}\label{field-run-on-sensible-load-01}
+
+This alpha field specifies if the coil system will operate to meet a sensible load calculated from the air flow rates through the coil system, coil system entering air temperature and coil outlet node (control node) temperature setpoint. There are two valid choices, Yes or No. If Yes, coil will run if there is a sensible load. If No, coil will not run if there is only a sensible load. The default is Yes.
+
+
+\paragraph{Field: Run on Latent Load}\label{field-run-on-latent-load-01}
+
+This alpha field specifies if the coil will operate to meet a latent load calculated from the air flow rate through the coil system, coil system entering air humidity ratio and coil system outlet node (control node) maximum humidity ratio setpoint. There are two valid choices, Yes or No. If Yes, the coil will run if there is a latent load. If both a sensible and latent load exist, the system will operate to maintain the temperature set point and then activate dehumidification control if needed. When only a latent load exists, the system will operate to meet the maximum humidity ratio set point and requires the use of a heating coil and heating coil outlet node air temperature set point manager downstream of this cooling coil to maintain the temperature set point. If No, the coil will not run if there is only a latent load. The default is No.
+
+
+\paragraph{Field: Minimum Air To Water Temperature Offset [deltaC]}\label{field-minimum-air-to-water-temperature-offset}
+
+The coil system will turn ON as required when coil entering air temperature is above coil entering water temperature by more than the amount of this offset [deltaC]. To model a waterside economizer connected to condenser loop an increased offset as desired. Default is 0.
```

Comment:
an increased offset *is* desired
OR
~an~ increase~d~ offset as desired

Reply:
Done.